pytorch_lightning/utilities/argparse_utils.py

```diff
@@ -160,7 +160,7 @@ def add_argparse_args(cls, parent_parser: ArgumentParser) -> ArgumentParser:
 
     allowed_types = (str, int, float, bool)
 
-    # TODO: get "help" from docstring :)
+    args_help = parse_args_from_docstring(cls.__doc__ or cls.__init__.__doc__)
```

Comment:
I think that we shall get help from init first as we preferably place arguments docs there
```suggestion
    args_help = parse_args_from_docstring(cls.__init__.__doc__ or cls.__doc__)
```